examples/notebooks_new_field/introduction.ipynb

```diff
@@ -0,0 +1,1099 @@
+    "where $\\nu\\leq\\frac{1}{\\sqrt{2}}$ and\n",
+    "\n",
+    "\\begin{equation}\n",
+    "h = \\min\\{ \\Delta x, \\Delta z \\}.\n",
+    "\\end{equation}\n",
+    "\n",
+    "The term $v_{max}$ is the maximum velocity propagation in $\\Omega$ and is given by:\n",
+    "\n",
+    "\\begin{equation}\n",
+    "v_{max}=\\max_{(x,z)\\in \\Omega}\\left\\vert c(x,z) \\right\\vert.\n",
+    "\\end{equation}\n",
+    "\n",
+    "The choice of $\\Delta t$ respecting the CFL condition defines the number of timesteps $NT$:\n",
+    "\n",
+    "\\begin{equation}\n",
+    "\\Delta t =\\frac{T}{NT}.\n",
+    "\\end{equation}\n",
+    "\n",
+    "In all the simulations we employ $\\nu=0.4 $. In this way, the temporal parameters are given by:\n",
+    "\n",
+    "- $\\Delta t$ $\\approx$ 0.0018 s = 1.8 ms;\n",
+    "- $NT$ = 10938.\n",
+    "\n",
+    " The source $f(x, z, t)$  will be a Ricker source with the following properties:\n",
+    "\n",
+    "- Postion at $x:$ $\\bar{x} = 35150 m = 35.15 Km$;\n",
+    "- Position at $z:$ $\\bar{z} = 32 m = 0.032 Km$;\n",
+    "- Peak frequence: $f_{0} = 5 Hz = 0.005 Khz$;\n",
+    "\n",
+    "The graph of $f(\\bar{x}, \\bar{z}, t)$ is given in the figure below.\n",
+    "\n",
+    "<img src='figures/source.png' width=800>\n",
+    "\n",
+    "We employ a synthetic velocity model $c(x,z)$, which represents a typical pre-salt section at Brazilian oil basins,  with the following properties:\n",
+    "\n",
+    "- Minimum propagation speed: $v_{min} = 1500 m/s = 1,5 Km/s$;\n",
+    "- Maximum propagation speed: $v_{max} = 7000 m/s = 7,0 Km/s$;\n",
+    "\n",
+    "The figure below shows the velocity profile.\n",
+    "\n",
+    "<img src='figures/velocity.png' width=800 height=800>\n",
+    "\n",
+    "We will consider 'receivers' along the $x$ direction (at all discrete points between $13.65$ Km and $56.65$ Km) at the depth $z=0.0032$ Km, where we measure the displacement $u(x,z,t)$ throughout the time integration. Based on these values at the receivers we bild a $seismogram$ at the end of the integration.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 1.5 - Numerical Simulations\n",
+    "\n",
+    "In the numerical simulations, we initially import the packages of interest, in particular, those that are part of the Devito library. First we import the basic Python packages."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy                   as np\n",
+    "import matplotlib.pyplot       as plot\n",
+    "import math                    as mt\n",
+    "import matplotlib.ticker       as mticker    \n",
+    "from   mpl_toolkits.axes_grid1 import make_axes_locatable\n",
+    "from   matplotlib              import cm"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We now import all the Devito package to have more freedom of work."
```

Comment:
This should be avoided and load only the necessary ones.

Reply:
The objective is if a new user would like to make new changes and/or add new structures all the packeges are avaible.